blocks/footer-block/features/footer/footer.scss

```diff
@@ -140,6 +140,8 @@ footer {
 
     .footer-header {
       font-weight: bold;
+      font-size: 14px;
+      line-height: 21px;
```

Comment:
@vinnie-olsen could we use the `calculateRem` function here?

Reply:
@beltrancaliz ah yep! I will make that change 👍

Reply:
done!